qiskit_ibm_runtime/base_runtime_job.py

```diff
@@ -371,7 +371,7 @@ def usage_estimation(self) -> Dict[str, Any]:
     @property
     def instance(self) -> Optional[str]:
         """For ibm_quantum channel jobs, return the instance where the job was run.
-        For ibm_cloud, `None` is returned.
+        For ibm_quantum_platform, `None` is returned.
```

Comment:
The behavior for `ibm_cloud` should still be documented until the option is removed.